sklearn/utils/__init__.py

```diff
@@ -589,3 +589,34 @@ def is_scalar_nan(x):
     # Redondant np.floating is needed because numbers can't match np.float32
     # in python 2.
     return bool(isinstance(x, (numbers.Real, np.floating)) and np.isnan(x))
+
+
+def init_arpack_v0(size, random_state):
```

Comment:
Thanks! Let's make it private (`_init_arpack_v0`)

Reply:
All right I'll do it, sorry my understanding was that private function was for function that was not supposed to be used outside of their module (this is one is used outside of utils) but maybe it is for function not to be used outside of sklearn in this case ?

Reply:
If it is part of the public API, we cannot changed/modify it without a deprecation cycle (for 2 major releases) as it could break users's code. Private functions, can be used anywhere in scikit-learn and changed as needed.